src/xenia/ui/window.h

```diff
@@ -172,8 +172,9 @@ class Window {
   Loop* loop_ = nullptr;
   std::unique_ptr<MenuItem> main_menu_;
   std::string title_;
-  int32_t width_ = 0;
-  int32_t height_ = 0;
+  // GTK must have a default value here that isn't 0
+  int32_t width_ = 1280;
```

Comment:
Is this safe on Windows? It has:
```
  if (width != width_ || height != height_) {
    width_ = width;
    height_ = height;
    Layout();
  }
```
in WM_SIZE handling.

Reply:
I don't think this matters